### ISSUE TITLE
Customize+ Scaling Not Loading during /gpose, character select screen

### ISSUE THREAD

**Issue by Nandanabe** [XIV-Tools/CustomizePlus#220]
Working as intended during general game play, but changes are not reflected on character select screen or during /gpose.

**Comment by RelicOfDeath** [comment]
I'm also having the same issue even after the update unsure what to do about it

**Comment by StoiaCode** [comment]
disable only owned

**Comment by yudhawee** [comment]
yes my problem as well, mine was having problem on certain Race, such as Elezen. all the Body part are scaling perfectly, but when entering Gpose, especially to make them *ahem* naked, only the Chest part doesnt change, whats wrong with this

**Comment by StoiaCode** [comment]
Only owned is disabled I assume?

**Comment by yudhawee** [comment]
it is Disabled, because i never tamper with anything for my gpose, and i just found it today, that every race EXCEPT Midlander chest customizer isnt working, i tried every race yesterday, only when it was bare naked though, Customize+ chest when they are Clothed is working weirdly

**Comment by yudhawee** [comment]
i've been trying my way around everytime i login and i actually found out the Issue about it, well before the earlier patch it was fine. im not sure if the problem was from this or the IVCS, because i found out if when i use the IVCS body, every scale that you edit except Chest are not changing inside Gpose. i try to use the normal modded body such as Bibo / ReqRider Non-IVCS, it works fine in Gpose now, so yes, might want to look further into it
